include/psa/crypto.h

```diff
@@ -16,6 +16,14 @@
 #include "crypto_platform.h"
 #endif
 
```

Comment:
Thank you for submitting a fix. But a fix to what? Can you please edit the pull request description to describe what the problem is?